scripts/src/main/resources/scripts/environment-project.bat

```diff
@@ -9,17 +9,22 @@ if "%WORKSPACE%" == "" (
   set WORKSPACE=%MAIN_BRANCH%
 )
 
-if "%SETTINGS_PATH%" == "" (
-  set SETTINGS_PATH=%WORKSPACES_PATH%\%MAIN_BRANCH%\%SETTINGS_REL_PATH%
+if "%WORKSPACE_VS%" == "" (
+  set WORKSPACE_VS=%WORKSPACESVS_PATH%
 )
+
+set WORKSPACE_PLUGINS_PATH=%WORKSPACES_PATH%\%WORKSPACE%\.metadata\.plugins
+set SETTINGS_PATH=%WORKSPACES_PATH%\%MAIN_BRANCH%\%SETTINGS_REL_PATH%
```

Comment:
this line is the replacement for the removed part above. IMHO the variant with the if is better as it allows some flexibility for overriding.